src/particles/BeamParticleContainerInit.cpp

```diff
@@ -59,7 +59,7 @@ namespace
         arrdata[BeamIdx::ux  ][ip] = ux * speed_of_light;
         arrdata[BeamIdx::uy  ][ip] = uy * speed_of_light;
         arrdata[BeamIdx::uz  ][ip] = uz * speed_of_light;
-        arrdata[BeamIdx::w][ip] = weight;
+        arrdata[BeamIdx::w][ip] = std::abs(weight);
```

Comment:
I would clip instead of flip signs:
```suggestion
        arrdata[BeamIdx::w][ip] = weight > 0._prt ? weight : 0._prt;
```

Reply:
I just saw: the signature of this function should probably use `amrex::ParticleReal` args instead of `amrex::Real`? :)

Reply:
In the current implementation, users must specify a positive total charge for an electron beam. This PR ensures that, even if the user specifies a negative charge (which could make sense for a bunch of electrons) the code behaves as expected. So I think that, in this instance, it is more appropriate to take the abs rather than clipping. An alternative would be to assert that the specified charge is always positive (and update the doc accordingly).